### PR TITLE
Events are now split into 3 different severities, each with their own timer

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

Love it when I get stray cargo pod or high priority bounty for the 10th time in a row - this way severe events are guaranteed to happen on a timed basis as opposed to having a high chance of not actually happening. This way, mild events happen more often allowing for more interesting stuff to happen with them, and severe events are guaranteed to happen at some point in the round to introduce fun.

# Spriting

# Wiki Documentation

# Changelog

:cl:  
rscadd: Events are now split into 3 different severities, each with their own timer
/:cl:

### CODE REVIEW

**Comment by Sniblet** [comment]
why are flutes catastrophic and not blob

**Comment by Simplehorror** [comment]
> why are flutes catastrophic and not blob

when was the last time you saw a blob win

**Comment by MajManatee** [comment]
like yesterday

**Comment by Simplehorror** [comment]
> like yesterday

fine

**Comment by Simplehorror** [comment]
@Xoxeyos EXPLAIN YOURSELF

**Comment by Marmio64** [comment]
honestly don't know if 3 categories are needed, seems like too much of a guarantee for bad shit to always happen. I'd like to see it be just 2 categories, one for nuisance/flavor events like pool chem spill, aurora, and minor space dust, then one for anything that can cause real problems even if small: meteors, ninjas, blob, major space dust, mice, etc.

**Comment by ReddicusDragon** [comment]
what are the performance implications of this

**Comment by Simplehorror** [comment]
> what are the performance implications of this

none

**Comment by Bibby0110** [comment]
Not a big fan, events are supposed to be random.

**Comment by Bibby0110** [comment]
DNM so Jamie doesn’t break events by not reviewing the code

**Comment by Simplehorror** [comment]
> Not a big fan, events are supposed to be random.

events are random, my events are shuffle

**Comment by ReddicusDragon** [comment]
I kinda want to see this as a test merge

**Comment by JamieD1** [comment]
I hope this respects player counts and minimum times

File: code/controllers/subsystem/events.dm [hunk 1]
```diff
@@ -178,10 +221,5 @@ SUBSYSTEM_DEF(events)
 
 /datum/controller/subsystem/events/proc/toggleWizardmode()
 	wizardmode = !wizardmode
-	message_admins("Summon Events has been [wizardmode ? "enabled, events will occur every [SSevents.frequency_lower / 600] to [SSevents.frequency_upper / 600] minutes" : "disabled"]!")
```
Comment (ReddicusDragon) [inline code/controllers/subsystem/events.dm:181]: ??? Will the event not occur every x to y minutes now?

File: code/game/gamemodes/wizard/wizard.dm [hunk 2]
```diff
@@ -55,11 +55,7 @@
 	for(var/obj/item/phylactery/P in GLOB.poi_list) //TODO : IsProperlyDead()
 		if(P.mind && P.mind.has_antag_datum(/datum/antagonist/wizard))
 			return FALSE
-
-	if(SSevents.wizardmode) //If summon events was active, turn it off
```
Comment (ReddicusDragon) [inline code/game/gamemodes/wizard/wizard.dm:59]: why remove this?

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/wizard/wizard.dm (hunk 2 above)
Comment (Simplehorror) [reply code/game/gamemodes/wizard/wizard.dm:59]: was dumb

File: code/controllers/subsystem/events.dm (hunk 1 above)
Comment (Simplehorror) [reply code/controllers/subsystem/events.dm:181]: no, it just doesn't message admins since it doesn't have a meaning anymore

**Comment by Simplehorror** [comment]
> I hope this respects player counts and minimum times

yes

**Comment by Bibby0110** [comment]
Kinda defeats the point of random events

**Pull request closed without merging**